data/itsdevdeepak.json

```
@@ -1,19 +1,41 @@
 {
   "name": "Deepak",
-  "bio": "Fullstack Developer | Javascript/Typescript | Open Source",
+  "bio": "Web Alchemist 🧙‍♂️ | Crafting Digital Elixirs with JavaScript  🪄 | Exploring the Open Source Realm 💫",
+  "tags": ["Open Source", "Javascript", "Typescript", "Reactjs", "NextJs", "Nodejs"]
```

Please add the missing comma here. 

```suggestion
  "tags": ["Open Source", "Javascript", "Typescript", "Reactjs", "NextJs", "Nodejs"],
```